calcom/cal.com
# feat(app-store): Add Giphy app

## What does this PR do?

Add a Giphy app that allows users to search and select gif for each event type

Fixes #2473 

## Type of change


## How should this be tested?

- Install the Giphy app from App Store 
- Create a new event or use any existing event type 
- Click on "Show advanced settings" & scroll to the bottom 
- Press search on the Giphy button which should open a popup 
- On the popup type something in the search box and press search 
- You can use the next & prev icons to see more gifs
- After selecting one press `Save`
- the selected Gif should be visible on the event edit page 
- Press the update button on the page 
- Preview this event and schedule a time, the gif should be visible on the success page 


## Checklist

- I haven't added tests that prove my fix is effective or that my feature works

## Screenshots
<img width="1440" alt="Screenshot 2022-04-23 at 8 41 17 PM" src="https://user-images.githubusercontent.com/25253907/164913446-967b007e-6b42-46ba-b60c-f6a2d1c9bca0.png">
<img width="1439" alt="Screenshot 2022-04-23 at 8 47 41 PM" src="https://user-images.githubusercontent.com/25253907/164913452-cb6ae3ff-2b13-444a-864a-967070fae644.png">
<img width="1435" alt="Screenshot 2022-04-23 at 8 41 38 PM" src="https://user-images.githubusercontent.com/25253907/164913456-0a9804cf-f0c7-4e86-8a57-e6f442f76178.png">
<img width="1262" alt="Screenshot 2022-04-23 at 9 32 44 PM" src="https://user-images.githubusercontent.com/25253907/164913855-7cb3d455-b5c6-4112-af57-f858e233d56b.png">
